http-servers/plain-text-server.js

```diff
@@ -0,0 +1,13 @@
+const http = require('http');
+
+const server = http.createServer();
+
+server.on('request', (req, res) => {
+    const body = 'Hello World';
+    res.writeHead(200, {
+        'Content-Type': 'text/plain'
+    });
+    res.end(body);
+});
+
+server.listen(8000);
```

Comment:
For production projects - hosts, ports etc are moved to constants or configs for different environments. The simplest - const PORT = process.env.PORT || 3000;